apm-lambda-extension/main_test.go

```diff
@@ -199,10 +202,26 @@ func initMockServers(eventsChannel chan MockEvent) (*httptest.Server, *httptest.
 	}
 	if err = os.Setenv("ELASTIC_APM_DATA_RECEIVER_SERVER_PORT", fmt.Sprint(extensionPort)); err != nil {
 		extension.Log.Fatalf("Could not set environment variable : %v", err)
-		return nil, nil, nil, nil
+		return nil
+	}
+
+	t.Cleanup(func() { lambdaServer.Close() })
+	return &lambdaServerInternals
+}
+
+// TODO : Move logger out of extension package and stop using it as a package-level variable
+func newLogger(t *testing.T, logLevel string) {
+	if err := os.Setenv("ELASTIC_APM_LOG_LEVEL", logLevel); err != nil {
+		t.Fail()
 	}
+}
 
-	return lambdaServer, apmServer, &apmServerInternals, &lambdaServerInternals
+func newTestStructs(t *testing.T) (context.Context, chan MockEvent) {
```

Comment:
AFAICS, the returned context is never used in any calls to `newTestStructs`. Can we get rid of it?

Reply:
Fixed in ecdc5f0